spec/features/budget_transaction_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.feature 'BudgetTransactions', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'hamid', email: 'hdm@gmail.com', password: '123456', password_confirmation: '123456',
+                        id: 1)
+    @budget_transaction = BudgetTransaction.create(name: 'test', amount: 23, author_id: @user.id)
+    @group = Group.create(author_id: @user.id, name: 'test', icon: 'http://test.com')
+    @group_transaction = GroupTransaction.create(budget_transaction: @budget_transaction, group: @group)
+    sign_in @user
+    visit group_budget_transaction_path(@group, @budget_transaction)
+  end
+
+  context 'Get /Show' do
+    it 'header should contain the text "Transaction"' do
+      within('header') do
+        expect(page).to have_content('Transaction')
+      end
+    end
+
+    # rubocop:disable Layout/LineLength
+    it 'should show details about the transaction' do
+      expect(page).to have_content(@budget_transaction.name)
+      expect(page).to have_content(@budget_transaction.amount)
+      expect(page).to have_content("#{@budget_transaction.created_at.strftime('%d %B %Y')} At #{@budget_transaction.created_at.in_time_zone.strftime('%I:%M %p')}")
+    end
+    # rubocop:enable Layout/LineLength
+
+    it 'should have a button to delete the transaction' do
+      expect(page).to have_button('Delete Transaction')
+    end
+
+    it 'clicking delete transaction button should delete the transaction' do
+      budget_transactions_count = BudgetTransaction.count
+      click_button('Delete Transaction')
+      expect(BudgetTransaction.count).to eq(budget_transactions_count - 1)
```

Comment:
- [ ] Great job working on the project so far but I highly recommend fixing the failing tests here since it should pass to let your project work as expected so kindly fix them 🚀✨🔥


![image](https://github.com/hamidazim321/budget_app/assets/58553711/e4ac9224-31e0-42df-b621-4b08b5acb5d9)

Reply:
Hello @Reem-lab, Can you provide the errors you are getting on the tests because I think it may be related to something else, maybe `rspec-rails` gem hasn't been installed properly or something else (from your screenshots only feature tests are failling) because all tests are passing correctly when I run them. Here is a screenshot of the results of tests I just ran now.
![s](https://github.com/hamidazim321/budget_app/assets/129533107/539da625-fd8b-432e-bea4-801f569ab7a3)